keyboards/kbd8x/rules.mk

```diff
@@ -62,6 +62,7 @@ SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 # if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
 NKRO_ENABLE = no            # USB Nkey Rollover
 BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE ?= yes      # Enable keyboard RGB underglow
```

Comment:
Why the question mark?!?

But seriously, it's not needed. And it's not a big deal.